metadata-ws/src/test/resources/features/common.feature

```diff
@@ -0,0 +1,100 @@
+Feature: Common object
```

Comment:
I am going to suggest to rename `common.feature` to `misc.feature` or something like that. And separate the functions only used for `misc.feature` to `MiscSteps` class or similar. The original idea of `Common` was for the functions shared by more than one object types.

Also rename this `Feature` to something like `Miscellaneous fuctions` would be more appropriate, imho.